docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md

```diff
@@ -0,0 +1,121 @@
+
+## Steps to Register a Foreign Asset
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the `Asset Hub` parachain
+
+      - For the local environment, connect to Local Node (Chopsticks), check the [Test Environment Setup](#test-environment-setup) section for more details
+      - For the live networks, connect to the `Asset Hub` parachain. You can use either Polkadot, Kusama, Rococo, or Paseo Asset Hub
+
+2. Click on the **Developer** tab on the left sidebar and select the **Extrinsics** section
+
+    ![Access to Developer Extrinsics section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-1.webp)
+
+3. Select the **foreignAssets** pallet from the dropdown list and choose the **create** extrinsic
+
+    ![Select the Foreign Asset pallet](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-2.webp)
+
+4. Fill out the required fields and click on the copy icon to copy the **encoded call data** to your clipboard. The fields to be filled are:
+
+    - **id** - as this is a foreign asset, the ID will be represented by a Multilocation that reflects its origin. For this case, the Multilocation of the asset will be from the source parachain perspective:
+  
+        ```javascript
+        MultiLocation {parents: 1, interior: X1(Parachain(2006))};
+        ```
+
+    - **admin** - refers to the account that will be the admin of this asset. This account will be able to manage the asset, including updating its metadata. As the registered asset corresponds to a native asset of the source parachain, the admin account should be the sovereign account of the source parachain
+      
+        ??? note
+            The sibling account can be obtained through [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/){target=\_blank}
+
+            ![Get parachain sovereign account](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-4.webp)
+
+            Ensure that **Sibling** is selected and that the **Parachain ID** corresponds to the source parachain. In this case, since the guide follows the test setup stated in the [Test Environment Setup](./register-a-foreign-asset.md/#test-enviroment-setup) section, the **Parachain ID** is `2006`.
+
+    - **minBalance** - the minimum balance required to hold this asset
+
+    ![Fill out the required fields](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-3.webp)
+
+    ??? note
```

Comment:
I think this should either be a `!!! note`, or if you want it to be expandable, add a title that describes what is in the note, so here it could be something like `??? note "Example encoded call data"`

This same thought applies to the other `??? note`s on this page